UserGuide.md

```diff
@@ -404,22 +418,19 @@ carrot_cli test run --test_input 001_HC_Dragen_test.WGS4.json --eval_input 002_H
 
 #### <a name="debugging-failed-runs-faq"/> **Debugging Failed Runs / FAQ**
 
+When a run in CARROT fails, it can be either due to a failure of the test or evaluation workflow in Cromwell (indicated by a status of `testfailed` or `evalfailed` respectively), or due to an error in CARROT (indicated by a status of `carrotfailed`).
 
 ##### <a name="interrogating-cromwell"/> **Interrogating Cromwell**
 
-When a run in CARROT fails, often it has proceeded far enough to have a Cromwell run ID associated with it.  If this is the case, you can look directly at the Cromwell failure by interrogating the Cromwell server using the Swagger Web API or by using a tool like [cromshell](https://github.com/broadinstitute/cromshell).  If you choose to use cromshell you must ensure that the cromshell server configuration file in `~/.cromshell/cromwell_server.config` points to the cromwell server that CARROT is using (including the correct port on which to communicate).
+If the failure happened in Cromwell, you can look directly at the Cromwell failure by interrogating the Cromwell server using the Swagger Web API or by using a tool like [cromshell](https://github.com/broadinstitute/cromshell).  If you choose to use cromshell you must ensure that the cromshell server configuration file in `~/.cromshell/cromwell_server.config` points to the cromwell server that CARROT is using (including the correct port on which to communicate).
```

Comment:
If there's an easy way to print the server and port from the carrot_cli I'd include that command here.

Reply:
Ooo good idea

Reply:
Oh wait no, there isn't an easy way to do that

Reply:
I should add something like that